<!-- launch-prefix="gdb -ex run -args" -->
<launch>
  <!-- motion adaption node -->
  <node pkg="robosem_motion_retargeting_controller" type="robosem_mo_ret_controller" name="robosem_mo_ret_controller">
    <!-- load parameters for the nodes onto the parameter server into the private namespace -->
<!--     <rosparam command="load" file="$(find motion_retargeting_controller)/config/robosem_head_adaption_config.yaml" /> -->
    <rosparam command="load" file="$(find robosem_motion_retargeting_controller)/config/robosem_upper_body_retargeting_config.yaml" />
    <!-- topic remappings -->
    <remap from="joint_states" to="/robosem/joint_states" />
    <remap from="action_server_name" to="/robosem/upper_body_controller" />
  </node>
</launch>
